Codegen/src/Nodes/TypedDataNode.cs

```diff
@@ -986,7 +986,23 @@ public static List<ITextLine> InsertChildren(ColumnsLayout? layout, List<string>
                 {
                     if (child.IsFlagSet(Flag.IsTypedefCopyNode))
                     {
-                        lines.AddRange(child.Lines);
+                        var linesContent = child.CodeElement.SourceText.Split(new string[] { System.Environment.NewLine }, System.StringSplitOptions.None);                        //Indent the line according to its declaration
+                        foreach (string line in linesContent)
+                        {
+                            //Only the line containing copy can be badly indented. 
+                            string lineText;
+                            if (line.IndexOf("COPY", StringComparison.OrdinalIgnoreCase) >= 0)
```

Comment:
Does it works with this kind of code:
```cobol
       01 MyType2 typedef strict public.
           05  MyData value "COPY". COPY TESTCPY.  
```

Reply:
comlete TestTypeDefCopy with these statements.
https://github.com/TypeCobolTeam/TypeCobol/pull/1648/commits/37d27ab498b4f125c1de0768948140b5bc0232a0